tools/scripts/x86_fsp/qemu/qemu.sh

```diff
@@ -110,6 +110,7 @@ if [ "$ENABLE_TPM" = true ]; then
     killall swtpm || true
     sleep 1
     echo TPM Emulation ON
+    rm -rf /tmp/swtpm || true
```

Comment:
I seem to remember there was a reason not to remove the tpm here. Likely testing persistency across qemu.sh runs via this script. I would not do this silently. If needed for your test, perhaps add an option?